.github/workflows/pr-actions.yml

```diff
@@ -46,12 +46,12 @@ jobs:
       script: 'npm run lint'
 
   # Run Jest Unit Tests
-  test-unit:
+  test:
```

Comment:
Renamed this to test since it comprises multiple types now.

Reply:
My only concern is clarity about whether this runs the e2e tests. Not a hard sticking point, but I wonder what's an appropriate terminology. At work we use 'unit test' to describe tests of individual functions which require the backend to be booted, so calling the sails tests unit tests feels fairly natural to me. Would you say they're more backend integration tests?

Reply:
I was wondering that, too. I think it’s fair to say they’re both unit tests, we could just change the sails name to be something like test:unit-sails

Reply:
Per our discord convo, we'll go with test:unit for testing both and test:unit:server and test:unit:client for the separate categories of unit tests